frontend/packages/ux-editor/src/components/config/editModal/EditOptions.tsx

```diff
@@ -125,8 +115,20 @@ export function EditOptions({
         }
         name={`${component.id}-options`}
         value={selectedOptionsType}
-        variant='horizontal'
-      />
+        inline={true}
+      >
+        {Children.toArray(
+          component.type === 'RadioButtons' && [
```

Comment:
I think we should move this component-type check outside the Radio.Group. We do not need/want to render an empty Radio.Group if we do not have any RadioButtons to render. 😊 

We should not need to use Children.ToArray here. 😊